stable/xray/values.yaml

```diff
@@ -55,6 +55,14 @@ xray:
         type: {{ .Values.database.type }}
         driver: {{ .Values.database.driver }}
       {{- end }}
+      {{- if and (not .Values.rabbitmq.enabled) (not .Values.common.rabbitmq.connectionConfigFromEnvironment) }}
```

Comment:
I don't understand the relation between `connectionConfigFromEnvironment` and `rabbitmq.external`. Why are these mutually exclusive? Also, I think it is very likely that someone would set `connectionConfigFromEnvironment` to false but will not provide the external rabbit details. This can lead to poor UX

Reply:
@eldada  Need your comments here -

Reply:
@danielezer  `connectionConfigFromEnvironment` enabled implies we use JF_SHARED_RABBITMQ values for internal rabbitmq  , for external using  systemyaml